java/src/jmri/jmrit/beantable/LRouteTableBundle_ca.properties

```diff
@@ -25,7 +25,7 @@ InputTitle = Selecciona l'estat del Disparador d'entrada per a aquesta L'Itinera
 MiscTab = Alineaci\u00f3 / Blocatge / etc.
 MiscTabHint = Selecciona script i fitxer de so, sensors d'alineaci\u00f3, altres par\u00e0metes
 MiscTitle = Accions L'Itinerari adicionls
-LRouteType = Tipus de L\u00f2gica d'Itinerari
+L'ItinerariType = Tipus de L\u00f2gica d'Itinerari
```

Comment:
Also, never translate the word before the =
When you do, the whole line is not found by the program as it looks for that same unstranslated word in all the .properties files.
So this should stay ``LRouteType =``